### DISCUSSION TITLE
check out my profile<title>

### DISCUSSION THREAD

**Discussion by Manjunathhs-0003** [community/community#116445] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

hello there, i upload projects on python 
checkout my repos at <code>www.github.com/Manjunathhs-0003</code>

**Comment by queenofcorgis** [comment]
Hi @Manjunathhs-0003 , we’re glad you’re here 😀

Thanks for sharing your projects with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
